src/helpers/game_utils.ts

```diff
@@ -26,9 +26,10 @@ interface GroupMatchResults {
  * @param ignoredSongs - List of YouTube video IDs of songs to ignore
  * @returns a list of songs, as well as the number of songs before the filter option was applied
  */
-export async function getFilteredSongList(guildPreference: GuildPreference, ignoredSongs?: Set<string>, alternatingGender?: GENDER): Promise<{ songs: QueriedSong[], countBeforeLimit: number }> {
+export async function getFilteredSongList(guildPreference: GuildPreference, ignoredSongs?: Set<string>, genderOverride?: Gender): Promise<{ songs: QueriedSong[], countBeforeLimit: number }> {
+    const fields = ["song_name as name", "artist_name as artist", "link as youtubeLink", "publishedon as publishDate", "members", "id_artist as artistId", "issolo as isSolo", "members"];
```

Comment:
Do we need to select these extra fields?

Reply:
These fields are required in the tests